src/main/java/com/spring/boot/project/demo/controllers/ProductController.java

```diff
@@ -25,6 +26,21 @@
     public List<ProductDto> getMostCommentedProduct(
             @RequestParam(defaultValue = "1000") int numberOfProducts) {
         return productService.findMostCommentedProduct(numberOfProducts).stream()
-                .map(productMapper::convertToProductDto).collect(Collectors.toList());
+                .map(productMapper::convertToProductDto)
+                .collect(Collectors.toList());
+    }
+
+    @ApiOperation(value = "Show all food items.")
+    @GetMapping
+    public List<ProductDto> getAllProduct() {
+        return productService.findAll().stream()
+                .map(productMapper::convertToProductDto)
+                .collect(Collectors.toList());
+    }
+
+    @ApiOperation(value = "Show food items by id.")
```

Comment:
```suggestion
    @ApiOperation(value = "Get product by id.")
```
why food? all of those products are food?